department-of-veterans-affairs/abd-vro
# MCP-1677: re-enable security config

# Re-enable security

## Description



### What was the problem?

Security of the API routes has been temporarily disabled because it was breaking integration tests

### How does this fix it?

- Re-enabled security
- Enhanced integration tests to use authorization
- Made JSON objects less sensitive to missing fields.

### Jira Tickets


- [MCP-1677](https://amida.atlassian.net/browse/MCP-1677)

## How to test this PR

- Go to http://localhost:8080/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config#/
- Click on the padlock next to an API.  A modal will appear. Enter a valid authorization key 

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
